### PR TITLE
Draft security proposal numfocus

### PR DESCRIPTION
**Pull request by Carreau**
This is a draft, I'll be happy to change anything

See #6

### CODE REVIEW

File: nf-security-committee.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Security Subcommittee for Numfocus
+
+While NumFOCUS and all it's affiliated and sponsored project are focused on
```
Comment (rcthomas) [inline nf-security-committee.md:3]: Looks like an incomplete sentence, should you just drop the "while" here and adjust?

File: nf-security-committee.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+which is at odds with the typical open-source communication channels. Setting up
+specific communication channels for each subproject can be costly for projects
+in term of time invested for documentation and monitoring, especially for
+subprojects. 
+
+The multiple projects and different ways of inquiring about security can make it
+hard for stakeholder to find how to contact the relevant projects; especially
+when a question or vulnerability is related to multiple project interacting
+together. 
+
```
Comment (rcthomas) [inline nf-security-committee.md:16]: Some of these are maybe restating/expanding on the points listed:

- Discussions about best practices for security in general and at a project level are good to have as a community and in public.  People need to be educated and informed about security in a general sense and should be able to find resources they need deploy and run software securely.
- At the same time, developers need to coordinate activity in anticipation of and in response to software issues to do with security, in a time-sensitive fashion, and with discretion (it is just the reality).  Each project can do this on its own but the structures needed are basically the same from project to project.

In addition, I thought (restating but also expanding a bit maybe):

- Sharing structures (both infrastructure and human resources) for managing security across multiple projects can make it more manageable but also enable better propagation and awareness of issues across a set of projects that are often linked together or may encounter similar issues.
- A common structure may also promote a coherent approach to certain common security issues across projects, and forming strategies around those common issues actually can make running and deploying software securely easier ("these are NumFocus projects and they tend to do things this way...").

File: nf-security-committee.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  response. The response can be directly from the concerned subproject. The 
+  security subcommittee can directly provide a response if:
+
+    - The subcommittee members have sufficient expertise to provide a response 
+    - The subprojects maintainers do not reply to inquiries – in which the
+      matter should also be discussed with numfocus board.
+
+- Ignore any non-security enquiries, warn and ban repeated offender from
+  contacting security@numfocus.org as their discretion.
+- Maintain a https://nufocus.org/security.txt text document following standard
```
Comment (rcthomas) [inline nf-security-committee.md:40]: typo, nufocus -> numfocus

File: nf-security-committee.md (hunk 1 above)
Comment (willingc) [inline nf-security-committee.md:3]:
```suggestion
Typically, NumFOCUS and all its affiliated and sponsored projects are focused on
```

File: nf-security-committee.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+- Julia : security@julialang.org.
+- Matplotlib https://github.com/matplotlib/matplotlib/blob/master/SECURITY.md
+  matplotlib@numfocus.org
+- Jupyter / IPython (security@ipython.org)
+
+
+For the following I was not able to find how to contact.
+
+- Nteract ??
```
Comment (willingc) [inline nf-security-committee.md:101]:
```suggestion
- Nteract https://github.com/nteract/nteract/security/policy
```

**Pull request closed without merging**